src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/DefendantResponseNotificationService.java

```diff
@@ -74,19 +71,21 @@ private String getDefendantResponseIssuedEmailTemplate(Claim claim) {
         Party party = claim.getClaimData().getClaimant();
         Response response = claim.getResponse().orElseThrow(() -> new IllegalStateException("Response expected"));
 
-        if (isFullDefenceAndNoMediation(response)) {
+        if (isFullDefenceAndDispute(response)) {
             return getEmailTemplates().getDefendantResponseWithNoMediationIssued();
-        }
-
-        if (party instanceof Individual || party instanceof SoleTrader) {
+        } else if (party instanceof Individual || party instanceof SoleTrader
```

Comment:
This condition is meaningless as it is covering all the possible party types

Reply:
@sabahirfan Okay, should I remove the else NotificationException error handler too?

Reply:
yes please, It also make send to rename `getDefendantResponseIssuedToIndividual()` as this is not only for individuals now